frontend/lib/home/home_page.dart

```diff
@@ -91,15 +91,13 @@ class HomePageState extends State<HomePage> {
     return Scaffold(
       body: AppFlowsStack(appFlows: appFlows, currentIndex: _currentTabIndex),
       floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
-      floatingActionButton: _currentTabIndex == mapTabIndex
-          ? FloatingActionMapBar(
-              bringCameraToUser: (position) async {
-                await mapPageController?.bringCameraToUser(position);
-              },
-              selectedAcceptingStoreId: selectedAcceptingStoreId,
-            )
-          // Returning a Container() instead of null avoids animations
-          : Container(),
+      floatingActionButton: FloatingActionMapBar(
```

Comment:
Moving this check inside of `FloatingActionMapBar` was necessary to have enough space for the snackbar. Let me know if there are better ways to achieve this.

Reply:
hm i can not find an issue with the old implementation?
If i click change language and switch to map.
Snackbar appears. Button is moved a bit up.  Even i think its not a big issue if it overlaps for some seconds tbh
[Screen_recording_20240805_101121.webm](https://github.com/user-attachments/assets/1dd3c5aa-2d04-4d22-b5d9-f6a737d932bb)

Reply:
Without this, I just get the following error and no snackbar is shown:
![image](https://github.com/user-attachments/assets/6876109e-4b1f-4b88-a366-c9dbb0df7989)